app/frontend/src/components/Bar/ScoreBar.tsx

```diff
@@ -38,8 +38,10 @@ interface ScoreBarProps extends ContainerProps {
 
 export default function SearchResult({ value, children }: ScoreBarProps) {
   const classes = useStyles();
-  return (
+  return process.env.REACT_APP_IS_BETA_ENABLED ? null : (
```

Comment:
In prod this would be `false`, so would end up showing the bars which is the opposite of what we want? 🤔